lib/runner/extensions/event.command.js

```diff
@@ -284,7 +284,7 @@ module.exports = {
                 this.host.execute(event, {
                     id: executionId,
                     // debug: true,
-                    timeout: payload.scriptTimeout,
+                    timeout: payload.scriptTimeout || _.get(this.options, 'timeout.script'),
```

Comment:
Need to be _.has payload.scriptTimeout ? Payload : options

Reply:
add docs to payload

Reply:
add a todo to allow individual script level timeout

Reply:
need to follow timeout inheritence like that of request

```
timeout: _.min([
            _.get(options, 'timeout.request'),
            _.get(options, 'timeout.global')
        ]), // validated later inside requester
```

and maybe part config done in init of createContext of sandbox